doc/source/dev/conduct/report_handling_manual.rst

```diff
@@ -0,0 +1,227 @@
+to be strictly voluntary and no party can be pressured to participate. If the
+Committee suggests mediation, it should:
+
+* Find a candidate who can serve as a mediator.
+* Obtain the agreement of the reporter(s). The reporter(s) have complete
+  freedom to decline the mediation idea, or to propose an alternate mediator.
+* Obtain the agreement of the reported person(s).
+* Settle on the mediator: while parties can propose a different mediator than
+  the suggested candidate, only if common agreement is reached on all terms can
+  the process move forward.
+* Establish a timeline for mediation to complete, ideally within two weeks.
+
+The mediator will engage with all the parties and seek a resolution that is
+satisfactory to all.  Upon completion, the mediator will provide a report
+(vetted by all parties to the process) to the Committee, with recommendations
+on further steps.  The Committee will then evaluate these results (whether
+satisfactory resolution was achieved or not) and decide on any additional
+action deemed necessary.
+
+
+How the committee will respond to reports
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When the committee (or a committee member) receives a report, they will first
+determine whether the report is about a clear and severe breach (as defined
+below).  If so, immediate action needs to be taken in addition to the regular
+report handling process.
+
+Clear and severe breach actions
++++++++++++++++++++++++++++++++
+
+We know that it is painfully common for internet communication to start at or
+devolve into obvious and flagrant abuse.  We will deal quickly with clear and
+severe breaches like personal threats, violent, sexist or racist language.
+
+In what follows, we define a SciPy contributor as someone who has done any of
+the following, at least a week prior to the incident in question:
+
+* sent at least one substantial email to the mailing list;
+* submitted a substantial pull request;
+* submitted a substantial issue report.
+
+When a member of the Code of Conduct committee becomes aware of a clear and
+severe breach, they will do the following:
+
+* Immediately disconnect the originator from all SciPy communication channels.
+* Reply to the reporter that their report has been received and that the
+  originator has been disconnected.
+* If the originator appears to be a previous contributor, the moderator may
+  try to contact the contributor by some other means to check whether their
+  account has been hacked.
+* If the originator is in fact a previous contributor, and the contributor
+  wants to be reconnected to the SciPy channels, then the acting moderator may
+  consider some cooling off period, an agreement not to repeat the behavior,
+  and email moderation.  A previous contributor also has the right to an
+  appeal to the code of conduct committee.
+* In every case, the moderator should make a reasonable effort to contact the
```

Comment:
Consider replacing with:

```
* In every case, the moderator should make a reasonable effort to contact the
  originator, and tell them specifically how their language or actions
  qualifies as a "clear and severe breach".  The moderator should also say that, if the originator 
  believes this is unfair or they want to be reconnected to Scipy, they have the right to ask for a 
  review, as below, by the Code of Conduct Committee. The moderator should copy this
  explanation to the Code of Conduct Committee.  
* The Code of Conduct Committee will formally review and sign off on all cases where this 
  mechanism has been applied to make sure it is not being used to control ordinary heated 
  disagreement.
```

Reply:
The above is a rewrite of https://github.com/rgommers/scipy/pull/14#discussion_r145913874

Reply:
Do you mean to remove the whole distinction between a SciPy contributor and anyone else? If so there's more to remove, and I think it would improve the text.

Reply:
Yes, I guess the distinction isn't important now - I doubt whether anyone posting for the first time with nasty stuff is likely to want to go through the procedure.

Reply:
Agreed that this part is getting a bit long-winded.

Scipy -> SciPy

Reply:
Ralf - do you want me to strip this down in another PR - or are you OK to do that?

Reply:
I'll do it now.